deps/juggler-v3/test.js

```diff
@@ -0,0 +1,30 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-connector-db2
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+"use strict";
+
+const should = require("../../test/init");
```

Comment:
Shouldn't this be `require("should");` ?

In mysql , it requires should.

![image](https://user-images.githubusercontent.com/6864736/74351376-08b09200-4d85-11ea-85d8-8f4cbc54da34.png)

Reply:
^ this was in juggler-v3/test.js

Reply:
Same issue in juggler-v4/test.js

![image](https://user-images.githubusercontent.com/6864736/74352004-e3705380-4d85-11ea-9aa5-f3f0bd7ff23a.png)